index.html

```diff
@@ -0,0 +1,99 @@
+<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+<!DOCTYPE html>
+<head>
+  <h1 id="title">Amazon Survey Form</form></h1>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body id=".left">
+  <title id="title">
+    <h1>Amazon Survey Form</h1></title>
+<header id="main-header" class="main-header"></header>
+ 
+    <p id="description"> Personal Information, Please<p>
+<main>
+     
```

Comment:
Please Don't leave empty space lines, In my opinion, without blank lines, it would look cleaner and more readable.